### ISSUE TITLE
F.normalize() to support multiple dimensions

### ISSUE THREAD

**Issue by mega-optimus** [pytorch/pytorch#99125]
### Motivation: 
I frequently encounter use cases where a tensor needs to be normalized along multiple dimensions at once, but `F.normalize(x, dim)` only supports `dim` being int. 
### Proposal: 
It will be convienent to support multi-dimensional normalization.
Thank you !

cc @svekars @carljparker @albanD @mruberry @jbschlosser @walterddr @mikaylagawarecki

**Comment by jbschlosser** [comment]
Hey @mega-optimus, I believe this is already supported?

```python
import torch
import torch.nn.functional as F

x = torch.randn(3, 4, 5)
output = F.normalize(x, dim=(1, 2))
```

**Comment by mega-optimus** [comment]
> Hey @mega-optimus, I believe this is already supported?
> 
> ```python
> import torch
> import torch.nn.functional as F
> 
> x = torch.randn(3, 4, 5)
> output = F.normalize(x, dim=(1, 2))
> ```
Oh, you are right! Thanks!
I get this impression from the document, which is not up-to-date.
https://pytorch.org/docs/stable/generated/torch.nn.functional.normalize.html

**Comment by soulitzer** [comment]
Leaving this open to track updating the docs
@mega-optimus, would you be interested in sending a PR?

**Comment by mega-optimus** [comment]
> Leaving this open to track updating the docs @mega-optimus, would you be interested in sending a PR?

Happy to. Can you give some pointers e.g. where the source of doc is located?

**Comment by soulitzer** [comment]
Great! You can find the docs for `F.normalize` here

https://github.com/pytorch/pytorch/blob/36a95625daf002bddeb8347d3a353becd4be2f84/torch/nn/functional.py#L4662-L4689
